### ISSUE TITLE
📣 [Promote]: Create blog post about SPFx Toolkit manage capabilities 

### ISSUE THREAD

**Issue by Adam-it** [pnp/vscode-viva#364]
### 🎯 Aim 

The aim is to create a blog post that would describe all of the different SPFx Toolkit management capabilities. So mainly focusing on the first two views of the toolkit 
![image](https://github.com/user-attachments/assets/77422258-15b5-4410-ae89-8cc6f16271db)

The aim would be to describe:
- [login experience ](https://github.com/pnp/vscode-viva/wiki/5.3-Login-to-your-tenant-&-retrieve-environment-details)
- [environment details](https://github.com/pnp/vscode-viva/wiki/5.3-Login-to-your-tenant-&-retrieve-environment-details#environment-details) - so basic links from tenant and health incidents list
- app catalog list - what is present there, what are the manage options we have for each app like enable/disable, remove, upgrade ect.
- [GitHub Copilot Chat Participant](https://github.com/pnp/vscode-viva/wiki/8.-SPFx-Toolkit-GitHub-Chat-Participant) - the `/manage` chat command to retrieve basic data from tenant

Ideally, the blog post should be added to our community PnP blog: https://pnp.github.io/blog/

### 🙏 Help

Are you a regular user of SPFx Toolkit and you wanna help out but don't have any skills in developing VS Code extensions then this is a perfect issue for you! Help us out to promote the tool so that we may spread the word and reach more SPFx developers and help them rock 🤩

### Tip

Previous marketing things done are all grouped in this [wiki page](https://github.com/pnp/vscode-viva/wiki/2.-Marketing-resources). Adding them here for inspiration

**Comment by Sandeep-FED** [comment]
I’ll take this @Adam-it

**Comment by Adam-it** [comment]
> I’ll take this @Adam-it

All yours. 
Please base your blog on the latest (pre) release as some improvements to the to the `/magae` chat command or to the app catalog list view were merged just a few days ago. Also we could recheck with progress on this issue #350
which is an awesome manage capability and also include it in the blog post once it out. 
And of course mention that we are adding more and more in this area 
Please tag me over when the blog post is ready 🙏.
Thanks for your help in spreading the news. You Rock 👏🤩

**Comment by Adam-it** [comment]
@Sandeep-FED I was wondering if you could provide a status update also in this one? Do you need anything from my side to get unblocked or to move forward?

**Comment by Sandeep-FED** [comment]
Hey @Adam-it, this is also WIP. I will share the content with you in a couple of days for review!

**Comment by Adam-it** [comment]
> Hey [@Adam-it](https://github.com/Adam-it), this is also WIP. I will share the content with you in a couple of days for review!

Awesome. Thanks for the update

**Comment by Adam-it** [comment]
@Sandeep-FED did you manage to prepare a second attempt for this blog post? Do you need some help to get you unblocked?

**Comment by Adam-it** [comment]
@Sandeep-FED any update? Please 🙏

**Comment by Adam-it** [comment]
@Sandeep-FED since you already had an attempt on this blog post I was wondering if you managed to refactor what you had and you have something ready?

**Comment by Sandeep-FED** [comment]
Yeah Adam, I'm cooking the content hopefully it will be ready in two days 🙂

**Comment by Adam-it** [comment]
> Yeah Adam, I'm cooking the content hopefully it will be ready in two days 🙂 
> 

Thanks for the update

**Comment by Sandeep-FED** [comment]
Blog got merged and published today!

**Comment by Adam-it** [comment]
> Blog got merged and published today!

Awesome work. You Rock 🤩
